css-flexbox/align-items.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: calc(100% / 3);
```

Comment:
`flex-basis` is a child property, so it cannot be applied to the flex parent. It needs to be applied to the flex children